manifest.json

```diff
@@ -25,8 +25,8 @@
         "tabs",
         "background",
         "contextMenus",
-        "http://*/",
-        "https://*/",
+        "https://*.force.com/*",
```

Comment:
Makes sense, just also prevents people from using lightning out with the inspector. Problem for another day.